File: php/tests/test_profile_get.php
<?

require_once('../EpiDBClient.php');
require_once('config.php');

$key = $_key;

$client = new EpiDBClient($key);
$client->server = $_server;
$res = $client->profile_get($_user_id, '');

var_dump($res);

// vim: ts=4 sts=4 expandtab

File: python/tests/test_profile_get.py
#!/usr/bin/env python

import sys
sys.path += ['../']

from epidb_client import EpiDBClient
import config

key = config.key

client = EpiDBClient(key)
client.server = config.server
res = client.profile_get(config.user_id)

print res

# vim: ts=4 sts=4 expandtab

